db/migrate/20170421215941_create_submission_packets.rb

```diff
@@ -0,0 +1,9 @@
+class CreateSubmissionPackets < ActiveRecord::Migration[5.0]
+  def change
+    create_table :submission_packets do |t|
+      t.references :poem, foreign_key: true, index: true, null: false
+      t.references :submission, foreign_key: true, index: true, null: false
+      t.string :status
```

Comment:
If the status depends on values set on the parent, perhaps it should be calculated rather than stored in the database?

e.g.

```ruby
class SubmissionPacket < ActiveRecord::Base
  def status
    # calculate based on parent status
  end
end
```

This prevents you from having corrupt data such as having a status on parent change but then the corresponding change doesn't happen on the submission packet.